packages/engine-test-utils/src/__tests__/engine-server/markdown/__snapshots__/dendronPub.spec.ts.snap

```diff
@@ -294,6 +333,72 @@ VFile {
 }
 `;
 
+exports[`dendronPub frontmatter tags GIVEN enableFrontmatterTags: true THEN rendered when available: single tag, PUBLISHING 1`] = `
+VFile {
+  "contents": "<h1 id=\\"fmtags\\"><a aria-hidden=\\"true\\" class=\\"anchor-heading\\" href=\\"#fmtags\\"><svg aria-hidden=\\"true\\" viewBox=\\"0 0 16 16\\"><use xlink:href=\\"#svg-link\\"></use></svg></a>Fmtags</h1>
+<p>has fm tags</p>
+<hr>
+<h2 id=\\"tags\\"><a aria-hidden=\\"true\\" class=\\"anchor-heading\\" href=\\"#tags\\"><svg aria-hidden=\\"true\\" viewBox=\\"0 0 16 16\\"><use xlink:href=\\"#svg-link\\"></use></svg></a>Tags</h2>
+<ol>
+<li><a title=\\"Private\\" style=\\"color: brown\\" href=\\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\\" target=\\"_blank\\">first (Private)</a></li>
+</ol>",
+  "cwd": "<PROJECT_ROOT>",
+  "data": Object {},
+  "history": Array [],
+  "messages": Array [],
+}
+`;
+
+exports[`dendronPub frontmatter tags GIVEN enableFrontmatterTags: true THEN rendered when available: single tag, PUBLISHING 2`] = `
+VFile {
+  "contents": "<h1 id=\\"fmtags\\"><a aria-hidden=\\"true\\" class=\\"anchor-heading\\" href=\\"#fmtags\\"><svg aria-hidden=\\"true\\" viewBox=\\"0 0 16 16\\"><use xlink:href=\\"#svg-link\\"></use></svg></a>Fmtags</h1>
+<p>has fm tags</p>
+<hr>
+<h2 id=\\"tags\\"><a aria-hidden=\\"true\\" class=\\"anchor-heading\\" href=\\"#tags\\"><svg aria-hidden=\\"true\\" viewBox=\\"0 0 16 16\\"><use xlink:href=\\"#svg-link\\"></use></svg></a>Tags</h2>
+<ol>
+<li><a title=\\"Private\\" style=\\"color: brown\\" href=\\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\\" target=\\"_blank\\">first (Private)</a></li>
```

Comment:
Is the fact that these links are rendered private unrelated to this change? The snap before the PR has them rendered non-private.

Reply:
Not sure why the snap before PR had them as non-private, but in the test `THEN rendered when available: multiple tags: publishing`, I only create the note `has.fmtags`, and the tags that this note has are `first and `second`. Since both `tags.first` and `tags.second` don't exist, I think they _should_ be private?

Reply:
Just checked. all proc tests that have `ProcFlavor.PUBLISHING` are correctly rendering them to be private since I don't create the tag notes in any of the tests.

All test cases for `ProcFlavor.PREVIEW` won't have them rendered as private because we only do that for publishing.

Reply:
> Not sure why the snap before PR had them as non-private, 

Looks like before the PR, the proc that was used to test didn't pass in the proc flavor as publishing, thus making it default to `ProcFlavor.REGULAR`.

I've confirmed that rolling back to before the PR, passing in `ProcFlavor.PUBLISHING` to the proc, and running it as-is will update the snapshot so that it correctly renders tags `first` and `second` as private.